doc/src/site/sphinx/configuration/configuration_properties.rst

```diff
@@ -364,6 +364,14 @@ External backend configuration properties
 |                                                       |                | suffix "k", "m" or "g" (e.g.        |
 |                                                       |                | ``450k``, ``3m``)                   |
 +-------------------------------------------------------+----------------+-------------------------------------+
+| ``sspark.ext.h2o.external.backend.stop.timeout``      | ``10000ms``    | How long we wait before for proper  |
```

Comment:
`How long we wait before for proper external backend before we give up on confirmation from worker nodes.` Would it be possible to rephrase the sentence?

Reply:
Yup, done